js/nullish-coalescing-is-not-as-scary-as-it-sounds/nullish-coalescing-is-not-as-scary-as-it-sounds.md

```diff
@@ -0,0 +1,50 @@
+Nullish Coalescing Operator( **??** ) is one of the features of **ES2020**. It gives the ability to truly check the **nullish** values. It is another logical operator other than OR ( || ) , AND ( && ) and NOT ( ! ) operators.
+
+####**How it works?**
+
+The operator returns its **Right Hand Side (RHS)** operand when its **Left Hand Side (LHS)** operand is **null** or **undefined**, otherwise it will return the LHS operand.
+
+Let's see how it works through some examples:
+
+![Nullish-Coalescing-Operator-Ex-1](https://res.cloudinary.com/dyyr6kwla/image/upload/v1590838686/nullish-coalescing/nullish-example-1_xaqjn0.png)
+
+In the above example, the component Search has a prop [[searchBoxPlaceholder]].
```

Comment:
You can also use `searchBoxPlaceholder` here to add inline code

Reply:
Sorry, I did not understand about it @iamshadmirza

Reply:
Instead of brackets [[something]], you can use single ticks ` around a word to insert inline code.